synadm/cli/room.py

```diff
@@ -40,6 +40,16 @@ def join(helper, room_id_or_alias, user_id):
     helper.output(out)
 
 
+@room.command()
+@click.argument("room_alias", type=str)
+@click.pass_obj
+def resolve(helper, room_alias):
+    """ lookup room id from alias
```

Comment:
maybe you could add something that states that all servers that joined that room are displayed as well, but in the third line in the docstring so it does only display with `synadm room resolve -h` and not with `synadm room resolve`, keeping it short there is good.

Reply:
and yeah if you find the time a docstring stating Args: would be nice since only then it will render nicely in the autogenerated sphinx docs. Like here: https://github.com/JOJ0/synadm/pull/41/files#diff-21da8274044fdb3c233e057a58a867c3e934030edd102cbe0a79a61858386d20L194

Reply:
or maybe it could be simply:
```
synadm room getid 'alias'
synadm room getalias 'id'
```
or even
```
synadm room id 'alias'
synadm room alias 'id'
```